sickbeard/search.py

```diff
@@ -498,7 +498,7 @@ def searchProviders(show, episodes, manualSearch=False, downCurQuality=False, ma
         search_mode = curProvider.search_mode
 
         # Always search for episode when manually searching when in sponly
-        if search_mode == 'sponly' and (manualSearch is True or manualSelect is True):
+        if search_mode == 'sponly' and (manualSearch or manualSelect):
```

Comment:
Nothing really changed here? (manualSearch is True or manualSelect is True) == (manualSearch or manualSelect)